code/index.html

```diff
@@ -4,18 +4,68 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project Name</title>
-    <!-- dont forget to add a css file and link it here! -->
-    <!-- If you're using google fonts, that should be added here -->
+    <title>Coding Club for Girls</title>
+    <link rel="stylesheet" type"text/css" href="style.css">
+    <link rel='icon' href='Logo1.png' type='image/x-icon'/ >
   </head>
+
+ 
+
   <body>
-    <!-- 
-      Write your code here inside the body to make it appear in the browser. 
-      Below is an example on how to use an image tag. Copy it and add it in the body, but outside the comment tag if you want to see how it looks like on the website. Then you can change the size and other styling for it in the css file.
+    <header>
+      <div class="header">
+        <img src="Logo1.png" alt="Logo Coding Club">
+        <h1>Coding Club for Girls</h1>
+      </div>
+    </header>
+
+    <menu>
+      <a href="./index.html">Home</a>
+      <a href="./workshops.html">Workshops</a>
+      <a href="https://www.bfh.ch/codingclub">Über uns</a>
+      <a href="https://www.epfl.ch/education/education-and-science-outreach/fr/promotion-education-sciences/promotion-des-sciences/coding-club-for-girls/">Partner</a>
+    </menu>
+
+    <h2>Willkommen beim Coding Club!</h2>
+    
+    <div class="box1">
+      <img src="Alain Herzog - VD_4mai2019_Minitel_Microbit2_AH.jpg" alt="Coding Club">
+      <div class="text_right"
+      <p>
+        <h3>
+          Apps programmieren?<br>
+          Computerspiele designen?<br>
+          Webseiten gestalten?
+        </h3> 
```

Comment:
I think it might be better to have the `<h3>` outside (above) of the `<p>` element. So the `<p>` could just start on line 40.